pkg/initializer_v2/test/unit/model/test_model.py

```diff
@@ -0,0 +1,86 @@
+import runpy
```

Comment:
I would name this file `huggingface_test.py` where we are going to unit tests all functionality from the huggingface.py file.